app/Models/Competition.php

```diff
@@ -31,16 +31,19 @@ public function users()
     /**
      * Gets a CSV export of all the users in this competition.
      *
+     * @param \Gladiator\Repositories\UserRepositoryContract $repository
      * @return \League\Csv\ $csv
      */
-    public function getCSVExport()
+    public function getCSVExport($repository)
```

Comment:
@weerd said :no_good: to instantiating the repo in the model, and the internets agreed with me. this was the quickest solution i could think of :/